src/components/study-pane.js

```diff
@@ -771,6 +786,88 @@ const StudyPane = (props) => {
         network.useEquipment(equipments.lines);
     }, [network]);
 
+    function showLineMenu(line, x, y) {
+        setLineMenu({
+            position: [x, y],
+            lineToApply: line,
+            display: true,
+        });
+    }
+
+    function closeLineMenu() {
+        setLineMenu({
+            display: false,
+        });
+    }
+
+    function handleLockout(lineId) {
+        lockoutLine(studyUuid, lineId)
+            .then((response) => {
+                if (response.status === 304) {
+                    enqueueSnackbar(
+                        intl.formatMessage({ id: 'UnableToLockoutLine' }),
```

Comment:
Use displayErrorMessageWithSnackbar function to handle all errors ?

Reply:
Added displayInfoMessageWithSnackbar function to display messages